Bourreau/spec/boutiques/boutiques_tester_spec.rb

```diff
@@ -0,0 +1,389 @@
+#
+# Bourreau-side tests for the Boutiques framework, using a mock tool (boutiquesTestApp.rb).
+# Tests the generated cluster task by using it to create cluster commands that test against
+# the mock tool and its expected output for each set of input parameters.
+#
+
+# Required files for inclusion
+require 'rails_helper'
+require 'spec_helper'
+require_relative 'test_helpers'
+
+# Add helper methods for performing tests
+include TestHelpers
+
+# Testing the Boutiques framework on the Bourreau side
+describe "Bourreau Boutiques Tests" do
+
+  # Run before block to create required input files
+  before(:all) do
+    FileUtils.touch('c')  # For -C
```

Comment:
You can take a look to what is done in `userfiles_controller_spec`, in this one we also create a temporary file, and removed it at the end.